### PR TITLE
bring back referral page

### LINKED ISSUE #1877: Referrals not showing up
This information is being returned in the `/v4/account/profile` response, but isn't showing up on the page:

screen shot 2017-05-25 at 8 25 13 am

### PR DESCRIPTION
**Pull request by na3d**
Closes #1877 
screen shot 2017-05-26 at 12 08 18 pm

### CODE REVIEW

**Comment by mparke** [comment]
@na3d rebase and check console for TypeErrors:
screen shot 2017-05-30 at 2 36 42 pm

**Comment by na3d** [comment]
Done

**Comment by na3d** [comment]
I want to throw in the idea of re-organizing the stats like.  If anyone has an idea for fleshing that out more later, to make it pretty.  Feel free.
Pending: 2
Complete: 3
Total: 5
Credit: $60

File: src/profile/layouts/ReferralsPage.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        Referrals reward you when you refer people to Linode. If someone
+        signs up using your referral code, you'll receive a credit of
+        $20.00, so long as the person you referred remains an active
+        customer for 90 days.
+      </p>
+      <div className="row">
+        <div className="col-sm-12">
+          <hr />
+        </div>
+      </div>
```
Comment (eatonphil) [inline src/profile/layouts/ReferralsPage.js]: Would you drop this `<hr />` section here too? With few exceptions, we do not use these to separate elements visually in the manager.

**Pull request merged**